azext_hanaonazure/modules_sdk/models/monitoring_details.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.serialization import Model
+
+
+class MonitoringDetails(Model):
+    """Details needed to monitor a Hana Instance.
+
+    :param hana_vnet: ARM ID of an Azure Vnet with access to the HANA
+     instance.
+    :type hana_vnet: str
+    :param hana_hostname: Hostname of the HANA Instance blade.
+    :type hana_hostname: str
+    :param hana_instance_num: A number between 00 and 99, stored as a string
+     to maintain leading zero.
+    :type hana_instance_num: str
+    :param db_container: Either single or multiple depending on the use of
+     MDC(Multiple Database Containers). Possible values include: 'single',
+     'multiple'. Default value: "single" .
+    :type db_container: str or
+     ~azure.mgmt.hanaonazure.models.HanaDatabaseContainersEnum
+    :param hana_database: Name of the database itself.  It only needs to be
+     specified if using MDC
+    :type hana_database: str
+    :param hana_db_username: Username for the HANA database to login to for
+     monitoring
+    :type hana_db_username: str
+    :param hana_db_password: Password for the HANA database to login for
+     monitoring
+    :type hana_db_password: str
+    """
+
+    _attribute_map = {
+        'hana_vnet': {'key': 'hanaVnet', 'type': 'str'},
+        'hana_hostname': {'key': 'hanaHostname', 'type': 'str'},
+        'hana_instance_num': {'key': 'hanaInstanceNum', 'type': 'str'},
+        'db_container': {'key': 'dbContainer', 'type': 'str'},
+        'hana_database': {'key': 'hanaDatabase', 'type': 'str'},
+        'hana_db_username': {'key': 'hanaDbUsername', 'type': 'str'},
+        'hana_db_password': {'key': 'hanaDbPassword', 'type': 'str'},
+    }
+
+    def __init__(self, **kwargs):
+        super(MonitoringDetails, self).__init__(**kwargs)
+        self.hana_vnet = kwargs.get('hana_vnet', None)
+        self.hana_hostname = kwargs.get('hana_hostname', None)
+        self.hana_instance_num = kwargs.get('hana_instance_num', None)
+        self.db_container = kwargs.get('db_container', "single")
```

Comment:
The default behavior should be to support MDCs (since this is the standard as of HANA1.0 SPS12 and HANA2.0); so default should be `multiple` (or `true` if we go with a boolean `HanaMdc`).

Reply:
auto-generated code

Reply:
If you would like this changed, it needs to be changed in the swagger spec and not in this PR